aqua/emacs-mac-app/Portfile

```
@@ -73,7 +73,7 @@ if {${os.major} >= 11 && ${os.platform} eq "darwin"} {
 subport ${name}-devel {
     set emacs_version   27.2
 
-    version         20210530
+    version         20210913
     revision        0
```

"Bump the revision" means the `revision` value needs to be incremented. That means here (for emacs-mac-app-devel) *and* for emacs-mac-app above.

```suggestion
    version         20210530
    revision        1
```